ionoscloud/resource_server.go

```diff
@@ -453,9 +460,6 @@ func resourceServerCreate(ctx context.Context, d *schema.ResourceData, meta inte
 	nic := ionoscloud.Nic{
```

Comment:
@cristiGuranIonos shouldn't we also remove this? I see that below we use this `nic = getNicData(d, nicPath)`. What do you think?